ec2_security_groups_dumper/main.py

```diff
@@ -93,7 +93,9 @@ def as_dict(self):
 
 class Firewall(object):
 
-    def __init__(self):
+    def __init__(self, region=None, profile=None):
```

Comment:
Can you add a docstring and describe what the region and profile args are for? Also assert that they are of the correct type if not None.